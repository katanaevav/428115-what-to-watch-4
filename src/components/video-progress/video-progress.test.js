import React from "react";
import renderer from "react-test-renderer";
import VideoProgress from "./video-progress.jsx";

it(`Render video progress bar and duration time`, () => {
  const tree = renderer.create(
      <VideoProgress
        currentProgress = {12}
        duration = {100}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
