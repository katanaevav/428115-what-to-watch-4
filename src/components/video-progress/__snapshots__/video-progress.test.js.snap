// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render video progress bar and duration time 1`] = `
<div
  className="player__controls-row"
>
  <div
    className="player__time"
  >
    <progress
      className="player__progress"
      max="100"
      value={12}
    />
    <div
      className="player__toggler"
      style={
        Object {
          "left": "12%",
        }
      }
    >
      Toggler
    </div>
  </div>
  <div
    className="player__time-value"
  >
    1:40
  </div>
</div>
`;
